products/BellHybrid/apps/application-bell-settings/presenter/BedtimeSettingsPresenter.cpp

```diff
@@ -65,6 +64,7 @@ namespace app::bell_settings
     }
     void BedtimeSettingsPresenter::exitWithoutSave()
     {
+        this->stopSound();
```

Comment:
is the sound stopped properly if the changes are saved?

Reply:
Yes, it is being stopped properly and this has been tested - stopSound() invoked with the onVolumeExit callback guarantees that